diagnostic.md

```diff
@@ -140,7 +157,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+They will all align on top of one another.
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree)